packages/jaeger-ui/src/model/ddg/types.tsx

```diff
@@ -51,6 +53,42 @@ export type TDdgModel = {
   visIdxToPathElem: PathElem[];
 };
 
+export type TDdgStateEntry =
+  | {
+      state: typeof fetchedState.LOADING;
+    }
+  | {
+      error: ApiError;
+      state: typeof fetchedState.ERROR;
+    }
+  | {
+      model: TDdgModel;
+      state: typeof fetchedState.DONE;
+      styleStates: Map<number, number>;
+    };
+
+export type TDdgState = Record<string, Record<string, Record<string, Record<string, TDdgStateEntry>>>>;
+
+export enum StyleStates {
```

Comment:
`EStyleState`, per the naming conventions we discussed and singular instead of plural per convention.